jwst/datamodels/container.py

```diff
@@ -69,6 +70,10 @@ def __init__(self, init=None, **kwargs):
         super(ModelContainer, self).__init__(init=None, **kwargs)
 
         self._models = []
+        self.allowed_exptypes = None
```

Comment:
Style comment: Since `allowed_exptypes` is for `ModelContainer` only, it may be appropriate to make it an explicit keyword argument. This would help in both self-documentation and in not carrying too much in the `kwargs` list.

Reply:
Moved allowed_exptypes to a keyword argument.